File: Lib/Telerik/Examples/QuickStart/Map/Shapefile/BookingForm.vb
Imports Telerik.WinControls.UI

Namespace Telerik.Examples.WinControls.Map.Shapefile
	Partial Public Class BookingForm
		Inherits RadForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Function ShowForm(ByVal performance As String, ByVal numberOfTickets As Integer, ByVal totalPrice As Integer, ByVal isReservation As Boolean) As DialogResult
			Me.radLabelPerformance.Text = performance
			Me.radLabelTickets.Text = numberOfTickets.ToString()
			Me.radLabelTotalPrice.Text = String.Format("{0:C0}", totalPrice)
			Me.radButtonBuyReserve.Text = If(isReservation, "RESERVE", "BUY")

			Me.radTextBoxEmail.Text = "john.smith@gmail.com"
			Me.radTextBoxName.Text = "John Smith"
			Me.radTextBoxCardNumber.Text = "1234-5678-8765-4321"
			Me.radTextBoxExpirationDate.Text = "10/2030"

			Return Me.ShowDialog()
		End Function

		Private Sub radButtonBuyReserve_Click(ByVal sender As Object, ByVal e As EventArgs) Handles radButtonBuyReserve.Click
			Me.DialogResult = DialogResult.OK
			Me.Close()
		End Sub
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/Map/Shapefile/BookingForm.cs
using System;
using System.Windows.Forms;
using Telerik.WinControls.UI;

namespace Telerik.Examples.WinControls.Map.Shapefile
{
    public partial class BookingForm : RadForm
    {
        public BookingForm()
        {
            InitializeComponent();
        }

        public DialogResult ShowForm(string performance, int numberOfTickets, int totalPrice, bool isReservation)
        {
            this.radLabelPerformance.Text = performance;
            this.radLabelTickets.Text = numberOfTickets.ToString();
            this.radLabelTotalPrice.Text = string.Format("{0:C0}", totalPrice);
            this.radButtonBuyReserve.Text = isReservation ? "RESERVE" : "BUY";

            this.radTextBoxEmail.Text = "john.smith@gmail.com";
            this.radTextBoxName.Text = "John Smith";
            this.radTextBoxCardNumber.Text = "1234-5678-8765-4321";
            this.radTextBoxExpirationDate.Text = "10/2030";

            return this.ShowDialog();
        }

        private void radButtonBuyReserve_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
            this.Close();
        }
    }
}
